src/launchpad/actions/expressions.js

```diff
@@ -0,0 +1,57 @@
+const constants = require("../constants");
+const { generateKey } = require("../utils/utils");
+
+function evaluateInput(input) {
+  return async function ({dispatch, client}) {
+    if (!client) {
+      console.warn("No client");
+      return;
+    }
+
+    try {
+      const packet = await client.clientCommands.evaluate(input, {});
+      dispatch(addExpression(input, packet));
+    } catch (err) {
+      console.warn("Error when evaluating expression", err)
+    }
```

Comment:
I'm surprised evaluate throws. seems like a weird API. I'd expect the packet to have the error